trinity/protocol/les/normalizers.py

```diff
@@ -8,13 +8,22 @@
 from eth.rlp.headers import BlockHeader
 
 from trinity.protocol.common.normalizers import BaseNormalizer
+from trinity.rlp.block_body import BlockBody
 
 TResult = TypeVar('TResult')
 LESNormalizer = BaseNormalizer[Dict[str, Any], TResult]
 
 
+# Q: Shouldn't this be named GetBlockHeadersNormalizer?
 class BlockHeadersNormalizer(LESNormalizer[Tuple[BlockHeader, ...]]):
     @staticmethod
     def normalize_result(message: Dict[str, Any]) -> Tuple[BlockHeader, ...]:
         result = message['headers']
         return result
+
+
+class GetBlockBodiesNormalizer(LESNormalizer[Tuple[BlockBody, ...]]):
```

Comment:
Yeah, so for the same reason, the `Get` should be dropped here. This normalizer only acts on the response value, which is sent with the `BlockBodies` command.

Reply:
Ah got it